lib/std/array_list.zig

```diff
@@ -108,6 +108,33 @@ pub fn ArrayListAligned(comptime T: type, comptime alignment: ?u29) type {
             mem.copy(T, self.items[i .. i + items.len], items);
         }
 
+        /// Replace range of elements `list[start..start+len]` with `items`
+        /// grows list if `len < items.len`. may allocate
+        /// shrinks list if `len > items.len`
+        pub fn replaceRange(self: *Self, start: usize, len: usize, items: SliceConst) !void {
+            const after_range = start + len;
+            var range = self.items[start..after_range];
```

Comment:
This should be `const` instead of `var`. Also it looks like this will cause an out-of-bounds access if after_range is > items.len. Can you add a test case for that code path?

Reply:
I think this is just confusion between the `items` argument and `self.items` field. This isn't slicing the argument with `after_range`. I stared at this for about 10 minutes thinking it was and wondering why a test case I added was passing anyhow.

`items` for the slice argument was to follow the name used in insertSlice(). maybe it should just be renamed to `slice` or `new_items`.